UsefulStuff.Core/Features/CameraShortcuts.cs

```diff
@@ -60,9 +60,9 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
             public static float GetCameraMultiplier(string axis)
             {
                 if (Input.GetKey(KeyCode.LeftControl))
-                    return Input.GetAxis(axis) / 6f;
+                    return Input.GetAxis(axis) / (6f * HSUS.CameraMultiplierFactor.Value);
```

Comment:
Shouldn't this divide after multiplying by the correction factor? The way it is now the camera will move even slower in AI/HS2 when everything has a larger scale, so it'll act differently from the KK line of games.

Reply:
If I just did that, the slow down function would become a speed up function. But I do agree that the camera moves even slower with the larger scale which isn't the expected behavior. Thanks for catching this. This is why you're Manly and I'm just KKY 😁. I guess the underlying issue is that the vanilla camera movement speed in AI/HS2 is 10 times too slow to begin with.

Would this be ok?
```csharp
            public static float GetCameraMultiplier()
            {
                float _adjDeltaTime = Time.deltaTime * HSUS.CameraMultiplierFactor.Value;
                if (Input.GetKey(KeyCode.LeftControl))
                    return _adjDeltaTime / 6f;
                if (Input.GetKey(KeyCode.LeftShift))
                    return _adjDeltaTime * 4f;
                return _adjDeltaTime;
            }
```
Maybe I should make the `6f` and `4f` configurable as well. What do you think?

Reply:
I think it's fine.

You could have "FastCamSpeed" and "SlowCamSpeed" settings that replace the constant values (instead of dividing by 6f it should multiply by 0.6f in that case so that both settings have values in the same units). In this case you'd want to replace `Time.deltaTime * HSUS.CameraMultiplierFactor.Value;` with a hardcoded `* 10f` for AI/HS2 to not have to have different defaults for the settings.